section2/exercises/if-statements.js

```diff
@@ -43,7 +43,13 @@ Log to the console the appropriate age range
 
 var dogAge = 3;
 // Write your conditional here
-
+if (dogAge <= "2") {
+  console.log('The dog is a puppy!');
+} else if (dogAge <= 9) {
+  console.log('The dog is an adult!');
+} else if (dogAge >= 10) {
```

Comment:
Nice solution here!

ANother way to write is make this last block an `else` rather than `else if` with that condition; some may argue it's more readable since your brain doesn't have to do the work of interpreting that condition.